manifests/nfd-topology-updater-daemonset.yaml.template

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nfd-topology-updater
+  name: nfd-topology-updater
+  namespace: node-feature-discovery
+spec:
+  selector:
+    matchLabels:
+      app: nfd-topology-updater
+  template:
+    metadata:
+      labels:
+        app: nfd-topology-updater
+    spec:
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          image: k8s.gcr.io/nfd/node-feature-discovery:v0.6.0
+          name: nfd-topology-updater
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: ["ALL"]
+            readOnlyRootFilesystem: true
+            runAsUser: 0
+          command:
+            - "nfd-topology-updater"
+          args:
+            - "--kubelet-config-file=/podresources/config.yaml"
+            - "--podresources-socket=/podresources/kubelet.sock"
+            - "--sleep-interval=3s"
+            - "--watch-namespace=rte"
+            - "--server=nfd-master:8080"
+## Enable TLS authentication (1/3)
+## The example below assumes having the root certificate named ca.crt stored in
+## a ConfigMap named nfd-ca-cert, and, the TLS authentication credentials stored
+## in a TLS Secret named nfd-worker-cert
+#            - "--ca-file=/etc/kubernetes/node-feature-discovery/trust/ca.crt"
+#            - "--key-file=/etc/kubernetes/node-feature-discovery/certs/tls.key"
+#            - "--cert-file=/etc/kubernetes/node-feature-discovery/certs/tls.crt"
+          volumeMounts:
+            - name: kubelet-podresources-conf
+              mountPath: /podresources/config.yaml
+            - name: kubelet-podresources-sock
+              mountPath: /podresources/kubelet.sock
+            - name: host-sysfs
+              mountPath: /host/sys
```

Comment:
most examples and test cases use host-sys 

see: https://github.com/kubernetes-sigs/node-feature-discovery/search?q=host-sys